specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json

```diff
@@ -402,8 +402,127 @@
         },
         "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "Update": {
-            "$ref": "./examples/Update.json"
+          "SignalR_Update": {
+            "$ref": "./examples/SignalR_Update.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/SignalR/{resourceName}/restart": {
+      "post": {
+        "tags": [
+          "SignalR"
+        ],
+        "description": "Operation to restart a SignalR service.",
+        "operationId": "SignalR_Restart",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/SignalRServiceName"
+          }
+        ],
+        "responses": {

```

Comment:
model the final response as well. It would help the client getting the final response. So if the final response is 200, please add it to responses. And add the schema of the response if there is one.

Reply:
Currently 202 is the only response by design. This API will trigger a long-running backend operation which will complete asynchronously. That's why we didn't include 200/201. 200 might give the caller a wrong sense that the operation is completed but actually not.  As for 201, we actually don't create anything in this operation.

We saw the warnings while validating the spec. That's why we suppress it in the readme. Just because 200/201/204 all makes us feel strange

Reply:
added 204